test/fixture/validate-installed-global.js

```diff
@@ -0,0 +1,3 @@
+import test from '../../'
```

Comment:
> I thought we only ignored tests/fixtures because it might have some esnext stuff in it, while production might not?

Yes, that's why XO ignores them, because fixtures might not follow code style or anything.

> It would be nice if XO had an easy way to multiple configs (and a way to specify which got applied would depend on glob patterns).

Yup, I've been planning this for a long time, just haven't had the time to implement it yet.

https://github.com/sindresorhus/xo/issues/22